CHANGELOG.md

```diff
@@ -6,14 +6,19 @@
   * To run queries, use the `TEXT` operator: `realm.all<Book>().query("description TEXT \$0", "fantasy novel")`.
 
 ### Fixed
-* Fixed an issue where the generator would incorrectly consider a `DateTime` field a valid primary key.
+* Fix the query parser, it needs to copy a list of arguments and own the memory. This will prevent errors like getting a different result from a query, if the list is modified after its creation and before the execution of the query itself. In the worst case scenario, if the memory is freed before the query is executed, this could lead to crashes, especially for string and binary data types. (Core upgrade, since core v12.5.0)
+* Fixed a potential crash when opening the realm after failing to download a fresh FLX realm during an automatic client reset (Core upgrade, since core v12.3.0)
+* Access token refresh for websockets was not updating the location metadata (Core upgrade, since core v13.9.3)
+* Fix an out-of-bounds read in sectioned results when sectioned are removed by modifying all objects in that section to no longer appear in that section (Core upgrade, since core v13.12.0)
```

Comment:
```suggestion
```
We don't support sectioned results, so this change does not impact our users

Reply:
Ah, yes. It is about sectioned result.